<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOpen</class>
 <widget class="QDialog" name="DialogOpen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open File</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpenFile">
       <property name="toolTip">
        <string>Open File</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxIsQuran">
       <property name="text">
        <string>ملف قرءان كريم</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxQuran">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogOpen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogOpen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxIsQuran</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxQuran</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
